configs/katello/3.9.yaml

```diff
@@ -0,0 +1,270 @@
+  :puppet-pulp:
+    :branch: 5.5.0
+    :repo: https://github.com/Katello/puppet-pulp
+  :puppet-candlepin:
+    :branch: 5.1.1
+    :repo: https://github.com/Katello/puppet-candlepin
+  :puppet-certs:
+    :branch: 4.2.0
+    :repo: https://github.com/Katello/puppet-certs
+  :puppet-qpid:
+    :branch: 4.2.2
+    :repo: https://github.com/Katello/puppet-qpid
+  :puppet-foreman_proxy_content:
+    :branch: 7.2.0
+    :repo: https://github.com/Katello/puppet-foreman_proxy_content
+  :puppet-service_wait:
+    :branch: 1.0.4
+    :repo: https://github.com/theforeman/puppet-service_wait.git
+:ignores:
+  - 23322
+  - 22961
+  - 24127
+  - 24317
+  - 24341
+  - 23785
+  - 24309
+  - 22565
+  - 22808
+  - 22897
+  - 23036
+  - 23201
+  - 23109
+  - 23265
+  - 23176
+  - 22377
+  - 23284
+  - 22648
+  - 23541
+  - 22795
+  - 23925
+  - 23767
+  - 22376
+  - 23978
+  - 24070
+  - 23198
+  - 22754
+  - 23648
+  - 23283
+  - 23171
+  - 23946
+  - 23542
+  - 23032
+  - 23762
+  - 23786
+  - 23760
+  - 23160
+  - 24095
+  - 24223
+  - 24079
+  - 22947
+  - 23988
+  - 23287
+  - 23302
+  - 23696
+  - 23745
+  - 22967
+  - 22595
+  - 22972
+  - 23330
+  - 24070
+  - 22802
+
+:gpg_key: '10668686'
+:mash_scripts:
+ - katello-3.9-thirdparty-candlepin-rhel7
+ - katello-3.9-rhel7
+ - katello-3.9-client-rhel7
+ - katello-3.9-client-rhel6
+ - katello-3.9-client-rhel5
+ - katello-3.9-client-fedora27
+ - katello-3.9-client-fedora28
+ - katello-3.9-client-sles11
+ - katello-3.9-client-sles12
+:tags:
+  - name: katello-3.9-thirdparty-candlepin-rhel7
+    based_off: katello-thirdparty-candlepin-rhel7
+    arches:
+      - x86_64
+  - name: katello-3.9-thirdparty-pulp-rhel7
+    based_off: katello-thirdparty-pulp-rhel7
+    arches:
+      - x86_64
+  - name: katello-3.9-thirdparty-pulp-rhel6
+    based_off: katello-thirdparty-pulp-rhel6
+    arches:
+      - x86_64
+  - name: katello-3.9-thirdparty-pulp-rhel5
+    based_off: katello-thirdparty-pulp-rhel5
+    arches:
+      - x86_64
+  - name: katello-3.9-rhel7
+    based_off: katello-nightly-rhel7
+    helper_tags:
+      katello-3.9-thirdparty-rhel7: katello-thirdparty-rhel7
+      katello-3.9-rhel7-override: null
+      katello-3.9-client-rhel7: null
+    build_target: katello-3.9-rhel7-build
+    build_package_group_source_tag: katello-nightly-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      katello-3.9-rhel7-build:
+        katello-3.9-rhel7-override: 0
+        foreman-plugins-1.20-rhel7-override: 3
+        foreman-1.20-nonscl-rhel7: 4
+        foreman-1.20-rhel7: 10
+      katello-3.9-rhel7-override:
+        katello-3.9-rhel7: 0
+      katello-3.9-rhel7:
+        katello-3.9-thirdparty-rhel7: 0
+      katello-3.9-client-rhel7:
+        katello-3.9-rhel7: 0
+        katello-3.9-thirdparty-pulp-rhel7: 2
+    external_repos:
+      - epel-7
+      - centos-sclo-rh-rhel-7
+      - centos-sclo-sclo-rhel-7
+      - centos-7-server
+      - rhel-7.0-server
```

Comment:
I think we're in the process of dropping this.

Reply:
ACK - drop this and add `centos-7-server-updates` above the `centos-7-server`